src/ShadowDOM/wrappers/Range.js

```diff
@@ -20,23 +20,69 @@
 
   var OriginalRange = window.Range;
 
+  var ShadowRoot = scope.wrappers.ShadowRoot;
+
+  function isElement(node){
```

Comment:
I would be in favor of adding jscs linting/formatting to our contribution workflow. Thanks for updating range.js using it. Once this change lands, we can get a little more formal in our use of it.